parse_test.go

```diff
@@ -254,3 +257,58 @@ func TestParseConfigFile(t *testing.T) {
 		})
 	}
 }
+
+func TestParseConfigFileVia(t *testing.T) {
+	t.Parallel()
+
+	rootFS := flag.NewFlagSet("root", flag.ContinueOnError)
+	config := rootFS.String("config-file", "", "")
+	iFlag := rootFS.Int("i", 0, "")
+	sFlag := rootFS.String("s", "", "")
+
+	subFS := flag.NewFlagSet("subcommand", flag.ContinueOnError)
+	dFlag := subFS.Duration("d", time.Second, "")
+	bFlag := subFS.Bool("b", false, "")
+
+	subCommand := &ffcli.Command{
+		Name:    "subcommand",
+		FlagSet: subFS,
+		Options: []ff.Option{
+			ff.WithConfigFileParser(ff.PlainParser),
+			ff.WithConfigFileVia(config),
+			ff.WithIgnoreUndefined(true),
+		},
+		Exec: func(ctx context.Context, args []string) error { return nil },
+	}
+
+	root := &ffcli.Command{
+		Name:    "root",
+		FlagSet: rootFS,
+		Options: []ff.Option{
+			ff.WithConfigFileParser(ff.PlainParser),
+			ff.WithConfigFileFlag("config-file"),
+			ff.WithIgnoreUndefined(true),
+		},
+		Exec:        func(ctx context.Context, args []string) error { return nil },
+		Subcommands: []*ffcli.Command{subCommand},
+	}
+
+	err := root.ParseAndRun(context.Background(), []string{"-config-file", "testdata/1.conf", "subcommand", "-b"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if *dFlag != time.Hour {
+		t.Error(*dFlag)
+	}
+	if !*bFlag {
+		t.Error(*dFlag)
+	}
+	if *sFlag != "bar" {
+		t.Error(*sFlag)
+	}
+	if *iFlag != 99 {
+		t.Error(*iFlag)
+	}
+
+}
```

Comment:
```suggestion
	if want, have := time.Hour, *dFlag; want != have {
		t.Errorf("d: want %v, have %v", want, have)
	}
	if want, have := false, *bFlag; want != have {
		t.Errorf("b: want %v, have %v", want, have)
	}
	if want, have := "bar", *sFlag; want != have {
		t.Errorf("s: want %q, have %q", want, have)
	}
	if want, have := 99, *iFlag; want != have {
		t.Errorf("i: want %d, have %d", want, have)
	}
}
```

Reply:
don't really see the need for this, IMO it just adds noise but it's your package so I've made the change anyway

Reply:
Well, it did allow me to catch and fix the bug below :)

```go
	if !*bFlag {
		t.Error(*dFlag)
	}
```